### ISSUE TITLE
Post event survey feedback 

### ISSUE THREAD

**Issue by VictoriaODell** [open-austin/iced-coffee#278]
Just trying this out. Hope that we can get some insight from our members on what they like and don't like moving forward. 

<img width="450" alt="screen shot 2017-09-19 at 10 23 47 pm" src="https://user-images.githubusercontent.com/10767430/30625502-3c7fd324-9d89-11e7-820c-5f6d8c6ed493.png">

<br><br>

Link to event survey: 
[Post Event Survey Form ](https://docs.google.com/forms/d/e/1FAIpQLSfG0gzXoA9XemPAIIipH3FNLrtP6Qy12ULTtLeRabrz6zrmuw/viewform)

**Comment by werdnanoslen** [comment]
Cool, this'd be great to have for in-person surveys, and for remote surveys we'd get an even better response rate if we sent the question itself (not the survey link) in an email and collect responses through replies.

**Comment by VictoriaODell** [comment]
@lesliebarrientos and @happinessk 

Andy mentioned above making these for in person and I think y'all talked about this recently. We used to have some paper ones but we only got a few responses then. At another organization I lead we would leave dedicated time for people to fill out the survey. Like 3 minutes or so. Then we would collect them up and review together as a leadership team at the end of the meeting. We didn't have a social like Cenote afterward though.  

If we develop a process for collecting and inputting paper survey feedback then I think it would be a good addition and help us be a more responsive organization. 

We do have some [responses collected, though minimal, now. ](https://goo.gl/forms/CkpADS7IO0IyhMVl2)
